src/messages/MForward.h

```diff
@@ -63,18 +67,14 @@ struct MForward : public Message {
     }
   }
 
-  PaxosServiceMessage *get_msg_from_bl() {
-    bufferlist::iterator p = msg_bl.begin();
-    return (msg_bl.length() ?
-        (PaxosServiceMessage*)decode_message(NULL, 0, p) : NULL);
-  }
-
 public:
   void encode_payload(uint64_t features) {
     ::encode(tid, payload);
     ::encode(client, payload, features);
     ::encode(client_caps, payload, features);
-    payload.append(msg_bl);
+    // encode client message with intersection of target and source
+    // features.  this probably doesn't matter, but might.
```

Comment:
This is not a helpful comment. Say when it *does* matter rather than that it probably doesn't.